### PR TITLE
Edit article model

### PR DESCRIPTION
**Pull request by MAlngaawy**
here I handled article model elements,
1- used `React-markdown` package to markdown the body content
2 - edit the articleModel storybook ( add the same API )
3 -  fix the Blank page `BUG` 
4 -  add some styles to the articleModel CSS file

To test the article model View direct .. You can go to this link ( https://malngaawy.charity.m3ntorship.com/articles/5eaa1b013482057530be1bef )

### CODE REVIEW

File: src/components/ArticleModel/styles.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.markdown p img:not(:first-child) {
+  margin-left: 1%;
+}
+
+@media (max-width: 768px) {
+  .markdown p img {
+    width: 100%;
+    margin-left: 0 !important;
```
Comment (muubar) [inline src/components/ArticleModel/styles.css:38]: can we do this without using important ?

File: src/containers/Articles/index.jsx [hunk 2]
```diff
@@ -33,21 +33,22 @@ const Articles = ({
...
   }
 
-  if (dataError) {
-    return 'error';
+  if (loading) {
+    return 'Loading......';
   }
-  if (data && articlesData) {
+
+  if (data.length) {
```
Comment (muubar) [inline src/containers/Articles/index.jsx:44]: try not to assume that people know that a length of zero would coerce to false here, go with `data.length > 0` as it's more explicit. always try to be more explicit when writing your code

File: src/components/ArticleModel/styles.css (hunk 1 above)
Comment (MAlngaawy) [reply src/components/ArticleModel/styles.css]: yes I will avoid using important by adding one more CSS selector style ( specific for the  element)

**Comment by Tramsi** [comment]
LGTM ... 3aash ya ng3awy!!!

**Pull request merged**